src/Compilers/VisualBasic/Portable/xlf/VBResources.es.xlf

```diff
@@ -29,12 +29,12 @@
       </trans-unit>
       <trans-unit id="ERR_OverridingInitOnlyProperty">
         <source>'{0}' cannot override init-only '{1}'.</source>
-        <target state="new">'{0}' cannot override init-only '{1}'.</target>
+        <target state="translated">"{0}" no puede sobrescribir la propiedad "{1}" de solo inicio.</target>
```

Comment:
"init-only"  wasn't translated for other languages, but was translated here. Is it a mistake it should always be not translated?